Libplanet.Tests/Blockchain/Policies/BlockPolicyTest.cs

```diff
@@ -148,32 +144,30 @@ public void ValidateBlocks()
             Assert.Null(
                 policy.ValidateBlocks(
                     MineBlocks(
-                        new[] { (0, 0), (1, 1), (3, 2) }
-                    ),
+                        new[] { (0, 0), (1, 1), (3, 2) }).ToList(),
                     FixtureEpoch + new TimeSpan(3, 1, 0)
                 )
             );
             Assert.IsType<InvalidBlockDifficultyException>(
                 policy.ValidateBlocks(
                     MineBlocks(
-                        new[] { (0, 0), (1, 1), (3, 1) }
-                    ),
+                        new[] { (0, 0), (1, 1), (3, 1) }).ToList(),
                     FixtureEpoch + new TimeSpan(3, 1, 0)
                 )
             );
             Assert.Null(
                 policy.ValidateBlocks(
                     MineBlocks(
-                        new[] { (0, 0), (1, 1), (5, 2), (9, 1) }
-                    ),
+                        new[] { (0, 0), (1, 1), (5, 2), (9, 1) })
+                        .ToList(),
```

Comment:
Is this indentation intended?

Reply:
I used Rider's fix spacing, but it can be written in one line. I'll fix it.